### ISSUE TITLE
deep transverse fissures in inferomedial aspect of temporal lobes

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#7383]
**Preferred term label:**
deep transverse fissures in inferomedial aspect of temporal lobes

**Synonyms**

**Definition (free text, please give PubMed ID)**
deep transverse fissures in inferomedial aspect of temporal lobes. These can be seen on prenatal ultrasound, pre- and postnatal MRI or at autopsy.
[PMID: 19098178](https://www.ncbi.nlm.nih.gov/pubmed/19098178) (Fig. 3 and 4)
[PMID:24585534](https://www.ncbi.nlm.nih.gov/pubmed/24585534)

**Parent term (use hpo.jax.org/app)**
temporal lobe dysplasia (new term), under Abnormal cerebral morphology HP:0002060

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Thanatophoric dysplasia type I (https://www.omim.org/entry/187600)
Thanatophoric dysplasia type II (https://www.omim.org/entry/187601)

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062

**Comment by azankl** [comment]
also see #7384 for another temporal lobe abnormality, justifying the creation of the parent term 'temporal lobe abnormalities'

**Comment by pnrobinson** [comment]
@azankl isn't it this case that "deep transverse fissures in inferomedial aspect of temporal lobes" is  'temporal lobe dysplasia'?
It would only make sense to make separate terms if this feature can occur independently of 'temporal lobe dysplasia' otherwise it is probably better just to have the term 'temporal lobe dysplasia' (which I just created).

**Comment by azankl** [comment]
@pnrobinson I'd say 'deep transverse fissures in inferomedial aspect of temporal lobes' is a more specific type of 'temporal lobe dysplasia', so could be a child term of 'temporal lobe dysplasia'. But maybe that is getting too specific, so happy to just add 'temporal lobe dysplasia' if you prefer.

**Comment by pnrobinson** [comment]
I will add annotations to the above disease. If distinct types of temporal lobe dysplasia pop up we can make the terms more specific

**Comment by azankl** [comment]
@pnrobinson Interestingly, temporal lobe dysplasia also seems to be a feature of Hypochondroplasia (PMID: 23649205) and Achondroplasia (PMID: 29170271). So we should probably annotate these disorders with this term as well.
